src/extension.ts

```diff
@@ -22,7 +22,7 @@ export function activate(context: vscode.ExtensionContext) {
 				new vscode.Range(position.with(undefined, 0), position)
 			);
 
-			if (textBeforeCursor.indexOf(CSConfig.SEARCH_PHRASE_START) != -1 && textBeforeCursor[textBeforeCursor.length - 1] === CSConfig.SEARCH_PHRASE_END) {
+			if (CSConfig.SEARCH_PHRASE_START.includes(textBeforeCursor) && textBeforeCursor[textBeforeCursor.length - 1] === CSConfig.SEARCH_PHRASE_END) {
```

Comment:
I don't think `includes` works either. 

In JS, the `includes` function is checking if the "whole" value of`textBeforeCursor` exists in the `SEARCH_PHRASE_START` string array, not just part of `textBeforeCursor`

Reply:
I think what you two want would be

`textBeforeCursor.includes(CSConfig.SEARCH_PHRASE_START)`

The includes method has been overwritten in a string object, it doesnt act like in an array. In the context of a string it is used to see if a string contains something.

```
const string = "foo";
const substring = "oo";

console.log(string.includes(substring));
```

Console Output:
`true`

Reply:
This is still unrelevant though as we are changing to regex at this point and therefore this becomes irrelevant.